### DISCUSSION TITLE
embedding model for send_pictures

### DISCUSSION THREAD

**Discussion by kalle07** [oobabooga/text-generation-webui#5462] [Q&A]
is it possible to change the image recognition model
abd if HOW ???
where and which format is desired ?

**Comment by kalle07** [comment]
???

**Comment by kalle07** [comment]
tell me please

**Comment by Amit-Rohila33** [comment]
What specific information or guidance are you seeking regarding changing the image recognition model? Can you provide more details about the problem or challenge you're facing so that I can offer more targeted assistance?

**Comment by kalle07** [comment]
hey,

if i understand right, its not the the LLM model that recognize the picture, right ?

LLava
https://huggingface.co/models?search=Llava%20gguf

https://github.com/oobabooga/text-generation-webui/discussions/4230#discussion-5714108
